platform/ios/Makefile

```diff
@@ -16,6 +16,10 @@ else
 xcodebuild := xcodebuild
 endif
 
+.PHONY: build_core
+build_core:
```

Comment:
Naming nit: Normally you'd just call it `core`, since make commands read like "make core", not "make build core", that'd be redundant.